contributing.md

```diff
@@ -22,38 +18,81 @@ $ cd microsoft-authentication-library-for-js
 ```
 
-Now decide if you want your feature or bug fix to go into the dev branch
-or the master branch.  **All bug fixes and new features should go into the dev branch.**
-
-The master branch is effectively frozen; patches that change the SDKs
-protocols or API surface area or affect the run-time behavior of the SDK will be rejected.
+Now decide if you want your feature or bug fix to go into the dev branch or the master branch.  **All bug fixes and new features should go into the dev branch.**
 
-Some of our SDKs have bundled dependencies that are not part of the project proper.  Any changes to files in those directories or its subdirectories should be sent to their respective
-projects.  Do not send your patch to us, we cannot accept it.
+The master branch is effectively frozen; patches that change the SDKs protocols or API surface area or affect the run-time behavior of the SDK will be rejected.
 
-In case of doubt, open an issue in the [issue tracker][].
+Some of our SDKs have bundled dependencies that are not part of the project proper.  Any changes to files in those directories or its subdirectories should be sent to their respective projects. Do not send your patch to us, we cannot accept it.
 
-Especially do so if you plan to work on a major change in functionality.  Nothing is more
-frustrating than seeing your hard work go to waste because your vision
-does not align with our goals for the SDK.
+In case of doubt, open an [issue](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues).
 
+Especially do so if you plan to work on a major change in functionality.  Nothing is more frustrating than seeing your hard work go to waste because your vision does not align with our goals for the SDK.
 
-### BRANCH
+### Create a branch for your feature/patch
 
-Okay, so you have decided on the proper branch.  Create a feature branch
-and start hacking:
+Okay, so you have decided on the proper branch.  Create a feature branch and start hacking:
 
 ```
 $ git checkout -b my-feature-branch
 ```
 
-### BOOTSTRAP
+## Installation and Bootstrapping
+
+This repository is a monorepo for all packages maintained by the MSAL.js team. It contains public `npm` packages in the [`lib`](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib) folder as well as some additional libraries in the [`extensions`](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions) folder.
+
+If you would like to build a specific package, you can visit the `README.md` for that specific folder to review the build process. The instructions here are specific to the monorepo. 
 
 After you have cloned the repository, run `npm install` in the root folder. This will install the depedencies for all of the libraries and samples in the repo, create linkages between packages that have depedencies on each other, and run the build process for each of the libraries.
```

Comment:
```suggestion
After you have cloned the repository, run `npm install` in the root folder. This will install the dependencies for all of the libraries and samples in the repo, create links between packages that depend on each other, and run the build process for each of the libraries.
```

Reply:
done.